lints/lint_dsa_has_param.go

```diff
@@ -0,0 +1,70 @@
+// lint_dsa_has_param.go
```

Comment:
It has always been like this, but I couldn't remember why we did this. Would you want this removed for every file then?